packages/react-dom/src/events/SimpleEventPlugin.js

```diff
@@ -41,93 +43,96 @@ import getEventCharCode from './getEventCharCode';
  *       bubbled: 'onAbort',
  *       captured: 'onAbortCapture',
  *     },
- *     dependencies: ['topAbort'],
+ *     dependencies: [TOP_ABORT],
  *   },
  *   ...
  * };
- * topLevelEventsToDispatchConfig = {
- *   'topAbort': { sameConfig }
- * };
+ * topLevelEventsToDispatchConfig = new Map([
+ *   [TOP_ABORT, { sameConfig }],
+ * ]);
  */
-const interactiveEventTypeNames: Array<string> = [
-  'blur',
-  'cancel',
-  'click',
-  'close',
-  'contextMenu',
-  'copy',
-  'cut',
-  'doubleClick',
-  'dragEnd',
-  'dragStart',
-  'drop',
-  'focus',
-  'input',
-  'invalid',
-  'keyDown',
-  'keyPress',
-  'keyUp',
-  'mouseDown',
-  'mouseUp',
-  'paste',
-  'pause',
-  'play',
-  'rateChange',
-  'reset',
-  'seeked',
-  'submit',
-  'touchCancel',
-  'touchEnd',
-  'touchStart',
-  'volumeChange',
+type EventTuple = [TopLevelType, string];
+const interactiveEventTypeNames: Array<EventTuple> = [
+  [DOMTopLevelEventTypes.TOP_BLUR, 'blur'],
```

Comment:
It seems like at this point we might as well just write `[TOP_BLUR, 'onBlur']` and remove the string transformation below. `on*` across a list like this should gzip well. Can you try this? It would be nice to get rid of the concatenation and slicing/uppercasing here.

https://github.com/philipp-spiess/react/blob/af02875c73e85300495d986c66488cd5a30b5470/packages/react-dom/src/events/SimpleEventPlugin.js#L134

Reply:
If we use `on*` as the base name, we'd still have to slice and lowercase to create the event name used for the dispatch configurations, see here: 

https://github.com/facebook/react/pull/12629/files/af02875c73e85300495d986c66488cd5a30b5470#diff-67a781594a9123abc44160cea6eb65f7R145

Reply:
> It would change the `eventTypes` keys though.

That's what I meant, yes. I'll try that so we can see the impact. IIRC RN registers events using `top*` strings as the `eventTypes` keys.

Reply:
OK, let me know when you're ready for another review pass!

Reply:
After replacing all `eventTypes` keys with `on*` and clean up the slice + lowercasing mess in the SimpleEventHub, the bundle sizes (both regular and gzipped) grew a little again: 

```
react-dom.production.min.js  (UMD_PROD)

Prev Size: 99.03 KB
Current Size: 99.13 KB
Diff: +0.1%

Prev Gzip: 31.69 KB
Current Gzip: 31.72 KB
Diff: +0.1%
```

I went ahead and fixed ReactTestUtils as well but it turns out that there are still 14 other modules with failing unit tests. I'm not sure if it's worth to look into that further, what do you think?